remote/remote.go

```diff
@@ -2,6 +2,7 @@ package remote
 
 import (
 	"fmt"
+	"github.com/google/go-containerregistry/pkg/v1/static"
```

Comment:
nit: could you please group this line with the other `go-containerregistry` imports?